modin/pandas/test/test_io.py

```diff
@@ -504,6 +505,8 @@ class TestReadCSV:
     def test_read_csv_delimiters(
         self, make_csv_file, sep, delimiter, decimal, thousands
     ):
+        if IsExperimental.get():
+            pytest.xfail("Segmentation fault; see PR #2347")
```

Comment:
we should skip this test altogether, because marking it as `xfail` does not prevent it from running and crashing the whole testing run, which renders CI red anyway

Reply:
Ok, changed.